### PR TITLE
ansible: add httplib2 package as resource

### PR DESCRIPTION
**Pull request by L2G**
Ansible's core `uri` module depends on the `httplib2` package from PyPI.

### CODE REVIEW

File: Library/Formula/ansible.rb [hunk 1]
```diff
@@ -94,11 +95,17 @@ class Ansible < Formula
...
+
   def install
     ENV["PYTHONPATH"] = libexec/"vendor/lib/python2.7/site-packages"
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
 
-    res = %w[pycrypto boto pyyaml paramiko markupsafe jinja2]
+    res = %w[pycrypto boto pyyaml paramiko markupsafe jinja2 httplib2]
     res += %w[isodate xmltodict kerberos pywinrm] # windows support
     res += %w[six requests websocket-client docker-py] # docker support
     res += %w[pyasn1 python-keyczar] # accelerate support
```
Comment (xu-cheng) [inline Library/Formula/ansible.rb]: We could just use `resources.each do |r| ...` in blow. If all of resources are used in such way.

**Comment by brakon** [comment]
when will this be added??

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/23dc1cdf6dd565f95f3a004e7ad00aa288ba7cf7. Thanks @L2G!

Please note `revision` should typically go under the `sha256` rather than under the `homepage`. Otherwise :100:.

File: Library/Formula/ansible.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/ansible.rb]: Agree with Xu here. I let it slide in this PR, but in future there's no reason to not adopt `resources.each do |r|`.

**Comment by astorije** [comment]
Why was this added?
Does ansible's core rely on `httplib2` to work on a host, or is it just to have the `uri` module work as a `local_action`?

If the second option, then this sadly does not make it work:

```
failed: [default -> 127.0.0.1] => {"failed": true}
msg: httplib2 is not installed

FATAL: all hosts have already failed -- aborting
```

**Pull request closed without merging**